smpp/server.go

```diff
@@ -318,7 +318,10 @@ func StubHandler(s Session, m pdu.Body) {
 // as a fall-back
 func RouterHandler(s Session, m pdu.Body) {
 	bodyBytes, _ := json.Marshal(m)
-	logger.Server.WithFields(log.Fields{
+	l := log.WithFields(log.Fields{
+		"source": "server",
+	})
+	l.WithFields(log.Fields{
```

Comment:
I don't think creating a logger every time is the best solution. I am aware of the problem that there's no instance of logger available. Maybe just use the logger package in this case.